### PR TITLE
Update data and enums for Galakrond’s Awakening

### PR DESCRIPTION
**Pull request by utilForever**
This revision includes:
- Update Descent of Dragons Balance Changes (#379)
- Update data and enums for Galakrond’s Awakening (#383)
    - Change the number of cards: 8303 -> 8637
    - Create card generation files for 'YEAR_OF_THE_DRAGON'

### CODE REVIEW

**Comment by utilForever** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- Sources/Rosetta/CardSets/YoDCardsGen.cpp  2
- Includes/Rosetta/CardSets/YoDCardsGen.hpp  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/utilForever/RosettaStone/pullRequest?prid=4892513&bid=16202544)

File: Sources/Rosetta/Cards/Cards.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@ std::vector<Card*> Cards::m_cards;
 
 Cards::Cards()
 {
-    m_cards.reserve(8303);
+    m_cards.reserve(8637);
```
Comment (kozistr) [inline Sources/Rosetta/Cards/Cards.cpp]: 전체 카드 수를 변수로 빼 두면 어떨까요??

Comment (utilForever) [reply Sources/Rosetta/Cards/Cards.cpp]: 좋은 생각입니다. 반영하겠습니다.

File: Tests/UnitTests/Cards/CardsTests.cpp [hunk 2]
```diff
@@ -16,7 +16,7 @@ TEST(Cards, GetAllCards)
     const std::vector<Card*> cards = Cards::GetInstance().GetAllCards();
 
     ASSERT_FALSE(cards.empty());
-    EXPECT_EQ(cards.size(), 8303u);
+    EXPECT_EQ(cards.size(), 8637u);
```
[the remark this answers is not in the document]
Comment (utilForever) [reply Tests/UnitTests/Cards/CardsTests.cpp]: 여기는 반영해서 수정할 예정입니다.

File: Tests/PythonTests/test_cards.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 def test_all_cards():
 	cards = pyRosetta.Cards.all_cards()
 
-	assert len(cards) == 8303
+	assert len(cards) == 8637
```
[the remark this answers is not in the document]
Comment (utilForever) [reply Tests/PythonTests/test_cards.py]: Python API에도 상수 변수를 추가했습니다.

**Comment by utilForever** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- Includes/Rosetta/CardSets/YoDCardsGen.hpp  3
- Sources/Rosetta/CardSets/YoDCardsGen.cpp  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/utilForever/RosettaStone/pullRequest?prid=4892513&bid=16202544)

**Pull request merged**